force-app/main/default/layouts/Case-SA Request.layout-meta.xml

```diff
@@ -112,10 +112,6 @@
                 <behavior>Edit</behavior>
                 <field>Patient_Validated__c</field>
             </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>Patient_Validated_Date__c</field>
-            </layoutItems>
```

Comment:
@marklise could you bring this one back as well?

Reply:
Fixed in 38159b3.